xbmc/ContextMenus.h

```diff
@@ -18,22 +18,22 @@ struct CEjectDisk : CStaticContextMenuAction
 {
   CEjectDisk() : CStaticContextMenuAction(13391) {} // Eject/Load CD/DVD!
   bool IsVisible(const CFileItem& item) const override;
-  bool Execute(const CFileItemPtr& item) const override;
+  bool Execute(const std::shared_ptr<CFileItem>& item) const override;
```

Comment:
Please add `#include <memory>`to this header.

Reply:
done